File: Database Applications/StudentSystem/StudentSystem.Model/Homework.cs
namespace StudentSystem.Model
{
    using System;

    public class Homework
    {
        public int Id { get; set; }

        public string Content { get; set; }

        public string ContentType { get; set; }

        public DateTime DateTime { get; set; }

        public int StudentId { get; set; }
        public virtual Student Student { get; set; }
    }
}

File: Database Applications/JavaORM/StudentSystem/src/main/java/model/Homework.java
package model;

import javax.persistence.*;
import java.util.Date;

@Entity
@Table(name = "homeworks")
public class Homework {

    @Id
    @GeneratedValue
    private int id;

    @Column(nullable =  false)
    private byte[] content;

    @Column(nullable = false)
    private String contentType;

    private Date sentOn;

    @ManyToOne
    private Student student;

    @ManyToOne
    private Course course;
}
